glottolog3/__init__.py

```diff
@@ -56,6 +57,7 @@ def main(global_config, **settings):
     """ This function returns a Pyramid WSGI application.
     """
     settings.update(CFG)
+    settings['sqlalchemy.url'] = os.environ.get('GLOTTOLOG_DATABASE_URL');    
```

Comment:
Make sure to only update this setting if the environment variable exists, don't want to set this to null.